crates/matrix-sdk-crypto/src/identities/manager.rs

```diff
@@ -50,6 +50,11 @@ enum DeviceChange {
     None,
 }
 
+enum IdentityUpdate {
```

Comment:
I find it somewhat hard to keep all these structures straight in my head. Possibly a better name would help: `IdentityUpdateResult`? And/or some documentation.

Reply:
Yes good point, there is also another struct called IdentityUpdates. I renamed to IdentityUpdateResult